src/types/libdef/npm/node-fetch_v2.x.x.js

```diff
@@ -0,0 +1,111 @@
+declare module 'node-fetch' {
```

Comment:
I took it from latest available version of the libdef and without touching it it still works for our version of Flow.

We don't _really_ care though because we don't use `node-fetch` in our code directly.